deltacat/compute/compactor/model/table_object_store.py

```diff
@@ -22,6 +22,9 @@ def store_table(self, table: LocalTable) -> LocalTableReference:
     def get_table(self, table_like: LocalTableReference) -> LocalTable:
         pass
 
+    @abstractmethod
+    def get_table_reference(self, table_ref: Any) -> Any:
```

Comment:
Passing a `table_ref` and getting the `table_reference` back? This method doesn't seem to be doing anything.

Reply:
Removed this abstract method